projects/microphysics/experiments/prog-run-all-months.py

```diff
@@ -0,0 +1,25 @@
+import sys
```

Comment:
It would be nice to move this logic into argo (and//or add a MultiInitJob) too to keep the front-end "declarative". This is not needed now, but if we start doing this kind of experiments a few more times, I can see us running into trouble.

I've found it hard to reproduce results submitted by these .py scripts... which is why I moved to the `query-wandb.py reproduce` and `end_to_end.py` workflow. It's  hard to map runs in wandb to scripts in this folder. 

I've also thought about adding a templating layer to end_to_end.py to enable this kind of thing.

Reply:
That makes sense, I've mostly been using the configs as templates and then checking reproduce or git file history to do reruns.